packages/modeling/src/operations/booleans/scission.js

```diff
@@ -9,7 +9,7 @@ const scissionGeom3 = require('./scissionGeom3')
 /**
  * Scission (divide) the given geometry into the component pieces.
  *
- * @param {...Object} geometries - list of geometries
```

Comment:
Both "geometries" and "objects" are used in the booleans. I went with "objects" since they take both paths and geometries. But in general i don't like "objects" because an object can be anything (including js objects). Open to suggestions here.